build_tools/benchmarks/comparisons/mobilenetv2_fp32_commands.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import os
+from typing import Optional
+
+from common.benchmark_command import *
+from common.benchmark_command_factory import BenchmarkCommandFactory
+
+_DEFAULT_NUM_BENCHMARK_RUNS = 50
+_DEFAULT_NUM_THREADS = 1
+
+
+class TfliteMobilenetV2FP32(TFLiteBenchmarkCommand):
+  """ Specializes the benchmark command to use TFLite. """
```

Comment:
Why are there spaces at the beginning of these? I don't think that's the style we use (https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)